### PR TITLE
15.0 [MIG] ir_filters_multi_company

### PR DESCRIPTION
**Pull request by hhgabelgaard**

### CODE REVIEW

File: ir_filters_multi_company/security/ir_filters.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```
Comment (legalsylvain) [inline ir_filters_multi_company/security/ir_filters.xml:4]:
```suggestion
<odoo noupdate="1">
```

**Review by legalsylvain — COMMENTED** [review]
Hi @hhgabelgaard. 

Thanks for porting this module. A remark inline. LGTM, otherwise.

File: ir_filters_multi_company/security/ir_filters.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
```
Comment (legalsylvain) [inline ir_filters_multi_company/security/ir_filters.xml]:
```suggestion
<odoo>
```

Sorry for the noice : ignore that last comment. see @pedrobaeza remarks here : https://github.com/OCA/multi-company/pull/506#discussion_r1429125380

Comment (hhgabelgaard) [reply ir_filters_multi_company/security/ir_filters.xml]: OK, reverted

File: ir_filters_multi_company/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module add multi-company management to user-defined filters
```
Comment (legalsylvain) [inline ir_filters_multi_company/readme/DESCRIPTION.rst:1]: could you elaborate a little bit ? 
thanks !

File: ir_filters_multi_company/views/ir_filters.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <record model="ir.ui.view" id="view_ir_filters_tree">
+        <field name="model">ir.filters</field>
+        <field name="inherit_id" ref="base.ir_filters_view_tree" />
+        <field name="arch" type="xml">
+            <tree position="inside">
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+
```
Comment (legalsylvain) [inline ir_filters_multi_company/views/ir_filters.xml:26]: could you also add the field in the search view ? base.ir_filters_view_search
thanks !

**Review by legalsylvain — COMMENTED** [review]
some minor comments inline. otherwise, LGTM. Thanks !

**Pull request closed without merging**